test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -449,6 +449,32 @@ void TestNetworkJobs::testDownload() {
                                          ExitInfo(ExitCode::SystemError, ExitCause::NotEnoughDiskSpace), exitInfo);
         }
     }
+
+    // Empty file
+    {
+        const LocalTemporaryDirectory temporaryDirectory("tmp");
+        const SyncPath local0bytesFilePath = temporaryDirectory.path() / "0bytes.txt";
+        const RemoteTemporaryDirectory remoteTmpDir(_driveDbId, _remoteDirId, "testDownload0bytesFile");
+        std::ofstream ofs(local0bytesFilePath, std::ios::binary);
```

Comment:
```suggestion
        {std::ofstream{local0bytesFilePath};}
```
This closes the file automatically.

Reply:
Done here: https://github.com/Infomaniak/desktop-kDrive/pull/571/commits/815851e59aded5f1508cfc3d2f329b6bf69d8a38